docs/howto/use_podman.md

```diff
@@ -0,0 +1,46 @@
+# How to Develop Using Podman Instead of Docker
+
+Podman is a container engine that is compatible with Docker but does not require a daemon and can run containers rootless. If you want to use Podman for development instead of Docker, follow these steps:
+
+## 1. Install Podman
+
+Refer to the [official Podman installation guide](https://podman.io/getting-started/installation) for your operating system.
+
+## 2. Configure Docker Compatibility
+
+Podman provides a Docker-compatible CLI. You can setup Podman so it runs whenever the docker commands are run, [follow the migration guide](https://podman-desktop.io/docs/migrating-from-docker/managing-docker-compatibility)
```

Comment:
Could we explicitly mention that elastic-package subcommands work with podman configured this way?

Reply:
I've updated the document, more straight forward